index.html

```diff
@@ -0,0 +1,177 @@
+        <div class="details-presentation fadeInAnimation delay5">
+          <p class="schedule">2015.10.15(THU) ~ 16(FRI)</p>
+          <p class="location">@ National Museum of Karea, Art Center Nabi and more</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="events">
+      <div class="container">
+        <h2 class="section-title" id="events-section-title">Upcoming events</h2>
+        <span></span>
+        <div class="events-wrapper">
+          <article class="event-card">
+            <img src="images/pexels-brett-sayles-3532438.jpg" alt="">
+            <h3 class="event-title">Live Jazz Nights</h3>
+            <p class="event-description">Evenings of live music featuring local or regional jazz artists.</p>
+          </article>
+
+          <article class="event-card">
+            <img src="images/dance.jpg" alt="">
+            <h3 class="event-title">Swing Dance Classes</h3>
+            <p class="event-description">Step into the rhythm of yesteryears and let the music guide your feet.</p>
+          </article>
+
+          <article class="event-card">
+            <img src="images/pexels-elina-sazonova-1841591.jpg" alt="">
+            <h3 class="event-title">Wine Tasting and Jazz</h3>
+            <p class="event-description">Wine tasting and jazz events while listening to live music to create a complete sensory experience.</p>
+          </article>
+
+          <article class="event-card">
+            <img src="images/pexels-on-restaurant.jpg" alt="">
+            <h3 class="event-title">Movie Night and Jazz</h3>
+            <p class="event-description">Classic movies related to the world of jazz in your restaurant with a special menu inspired by the movie </p>
+          </article>
+        </div>
+        <button class="button mobile">Join Us for the Jazz Experience</button>
+        <a href="#" class="events-link desktop">SEE THE HOLE PROGRAM</a>
+      </div>
+    </section>
+
+    <section class="products">
+      <div class="container">
+        <h2 class="section-title">Featured Dishes</h2>
+        <span></span>
+        <div class="dishes-wrapper">
+          <!--
+          <article class="card-product">
+            <img src="images/first-dish.jpg" alt="">
+            <div class="product-description">
+              <h3>Seafood Symphony</h3>
+              <p>A medley of fresh seafood such as shrimp, mussels, and calamari in a rich white wine and herb sauce, served with freshly baked bread.</p>
+            </div>
+          </article>
+
+          <article class="card-product">
+            <img src="images/grilled-beef-steak-dark-wooden-surface.jpg" alt="">
+            <div class="product-description">
+              <h3>Steak Jazz</h3> 
+              <p>A juicy grilled steak accompanied by a red wine and mushroom reduction, served with mashed potatoes and seasonal vegetables.</p>  
+            </div>
+          </article>
+
+          <article class="card-product">
```

Comment:
- [ ] Kindly remove all the commented code in this file, this will ensure that you have a clean codebase. 👍